code/game/objects/items/extinguisher.dm

```diff
@@ -106,6 +106,7 @@
 		return 0
 
 /obj/item/extinguisher/afterattack(atom/target, mob/user , flag)
+	. = ..()
 	// Make it so the extinguisher doesn't spray yourself when you click your inventory items

```

Comment:
you're reintroducing this bug

Reply:
I'm not sure why calling a proc that previously did nothing would cause that bug to return so I pulled up Basil and my local game onside of eachother to test it and make sure and its functioning the same as it does on Basil. I'm clicking my inventory with an extinguisher on each and the extinguisher sprays when you click an item in your backpack but not when the item is equipped on both my game and on basil, but thats a separate bug.